src/test/java/com/hocs/test/glue/decs/SearchStepDefs.java

```diff
@@ -293,17 +293,17 @@ public void iSearchForTheCaseByItsCaseReference() {
         }
     }
 
-    @And("I search for the COMP case escalated to COMP2 by it's case reference")
+    @And("I search for the complaints case escalated to stage 2 by it's case reference")
     public void iSearchForTheEscalatedCOMPCaseByCaseReference() {
-        String compCaseRef = sessionVariableCalled("compCaseReference");
-        search.enterComplaintsSearchCriteria("Case Reference", compCaseRef);
+        String complaintCaseRef = sessionVariableCalled("complaintsCaseReference");
```

Comment:
Can we set this session variable to be a bit more descriptive? Something like "stage2ComplaintsCaseReference" would be good

Reply:
Updated to stage2CaseReference